src/main/java/com/suse/saltstack/netapi/datatypes/cherrypy/Applications.java

```diff
@@ -0,0 +1,117 @@
+package com.suse.saltstack.netapi.datatypes.cherrypy;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Date;
+
+public class Applications {
+
+    @SerializedName("Uptime")
+    private double uptime;
+
+    @SerializedName("Bytes Read/Second")
+    private double readsPerSecond;
+
+    @SerializedName("Current Time")
+    private Date currentTime;
+
+    @SerializedName("Server Version")
+    private String serverVersion;
+
+    @SerializedName("Total Time")
+    private double totalTime;
+
+    @SerializedName("Enabled")
+    private boolean enabled;
+
+    @SerializedName("Start Time")
+    private Date startTime;
+
+    @SerializedName("Bytes Written/Second")
+    private double writesPerSecond;
+
+    @SerializedName("Total Bytes Read")
+    private int totalBytesRead;
+
+    @SerializedName("Current Requests")
+    private int currentRequests;
+
+    @SerializedName("Total Requests")
+    private int totalRequests;
+
+    //TODO: implement requests
+    //@SerializedName("Requests")
+    //private Object requests;
```

Comment:
I was actually about to duplicate @moio but then I refrained (knowing he will do that anyway :-) ). Personally I am totally fine to have todos marking in the code (after all IDEs are marking them as tasks), as long as there is a new PR coming right away soon. It is indeed easier and much more comfortable to double-click in IDE a task, jumping to a TODO and implement it, rather than forget some issue in a GitHub through the hassle. But yes, I agree with @moio  about other comments. Better would be to move all the stuff to a GitHub issues or Gist etc, like:
`//TODO: Implement scary Godzilla, https://github.com/SUSE/saltstack-netapi-client-java/issues/666`